maps/src/main/java/com/gluonhq/maps/tile/TileRetriever.java

```diff
@@ -41,4 +41,5 @@
      * @return an image representing the tile
      */
     Image loadTile(int zoom, long i, long j);
+    String copyright();
```

Comment:
This method makes more sense in `OsmTileRetriever`.

Reply:
Are you sure? In this case MapView would have to accept a `OsmTileRetriever` instead of `TileRetriever` as its constructor argument.

Reply:
Apologies for the delay. I somehow missed the notifications.

I am not sure. @jperedadnr do you have a preference here? Will all types of TileRetriever need a copyright license and it make sense to add "copyright" to `TileRetriever` interface?

Reply:
I've done a full review of the PR, looks good to me, with some comments that should be addressed.

About the copyright, yes, it might not be mandatory, but I'd say it can stay with `TileRetriever`. 

In my comments, I suggest to use CSS for the label, in which case you can set its visibility to false. Even setting the copyright text to null won't show it. (We don't really have a way to enforce the developer to use/show the license).